src/components/Channel/Channel.tsx

```diff
@@ -781,54 +785,113 @@ const ChannelInner = <
   const jumpToFirstUnreadMessage = useCallback(
     async (queryMessageLimit = 100) => {
       if (!(client.user && channelUnreadUiState?.unread_messages)) return;
```

Comment:
```suggestion
      if (!channelUnreadUiState?.unread_messages) return;
```
Can we expect `client.user` to be always available at this point in time? I don't see it being used anywhere within the scope of the function so I feel like it'd be safe to remove it if it's not being used for type narrowing.

Reply:
Removed client as dependency from the memoized function.